chart/openfaas/templates/queueworker-dep.yaml

```diff
@@ -21,11 +21,13 @@ spec:
       - name:  queue-worker
         image: {{ .Values.queueWorker.image }}
         imagePullPolicy: {{ .Values.openfaasImagePullPolicy }}
-        {{- if .Values.functionNamespace }}
         env:
+        {{- if .Values.functionNamespace }}
```

Comment:
This is still wrong. the if statement is checking if a custom function namespace is given and should only set a value for `faas_function_suffix` and nothing else